Lesson 3 - Homework was added

BattlePancake opened the pull request:
4 tasks were added.

BattlePancake committed:
Lesson 3 - Homework was added

VitalyOv commented on TestProject_5/Program.cs:
```diff
@@ -1,2 +1,195 @@
+//         result = operand1 % operand2;
+//         Console.WriteLine($"Your calculation is: {operand1} % {operand2} = {result}");
+//         break;
+// }
```
Добавьте секцию default для ситуации когда ввели операцию которой нет в программе.

BattlePancake replied:
@VitalyOv 'default' section was added.

VitalyOv commented on TestProject_5/Program.cs:
```diff
@@ -1,2 +1,195 @@
+// double operand1 = double.Parse(Console.ReadLine());
+// Console.Write("Please, choose your operation from available: \n+   -   *   /   %\n\n");
+// string sign =Console.ReadLine();
+// if (sign != "+" && sign != "-" && sign != "*" && sign != "/" && sign != "%")
```
А это можно убрать и перенести вывод ссобщение в default

BattlePancake replied:
@VitalyOv 'if' section was replaced by 'default'

VitalyOv commented on TestProject_5/Program.cs:
```diff
@@ -1,2 +1,195 @@
+// string num2 = num1.Substring(num1.Length - 1);   //Нашёл функцию 'Substring', которая позволяет
+// switch(num2)                                     //забрать последнее число из строки для сравнения
+// {
+//     case "0":
```
Интересный путь) Вот так еще можно было - Если поделить число на 2 с типом double то у него будет дробная часть(если это нечетное). Приводим результат к int у нас отбрасывается. double - int . получаем в double дробную часть. 5.5 - 5 = 0.5 и если он не равен 0 то число нечетное.

BattlePancake replied:
@VitalyOv added this as a 3rd way

BattlePancake committed:
Lesson 3 - Homework - Update №1

1st Task: 'default' section was added instead of 'if', 4th Task: 3rd way was added
